setup/installation.md

```diff
@@ -17,6 +17,14 @@ Install the following dependencies if you haven't already:
 [PostgreSQL](https://www.postgresql.org/docs/12/admin.html) instance to store
 data, including workspace information and session tokens.
 
+## Doctor
+
+Before proceeding, we strongly recommend running Doctor, which is Coder-supplied
```

Comment:
```suggestion
Before proceeding, we strongly recommend running [Doctor](https://github.com/cdr/coder-doctor), which is Coder-supplied
```

Reply:
@johnstcn this repo is currently private; will we be making it public?

Reply:
As far as I know, that's the plan!

Reply:
I think we should probably include instructions on how to download from GitHub, unzip, and run Doctor. We can either do this on the doctor repo, or in docs.

Rationale is that our docs include all helm commands, etc. Install steps for Doctor will be helpful